front/src/Api/Events/TagEvent.ts

```diff
@@ -0,0 +1,10 @@
+import * as tg from "generic-type-guard";
+
+export const isTagEvent =
+    new tg.IsInterface().withProperties({
+        list: tg.isArray(tg.isString),
+    }).get();
+/**
+ * A message sent from the iFrame to the game to show/hide a layer.
```

Comment:
Fix this.
Also, I think we don't need a specific event for this. We could use the "GameStateEvent" to fetch tags.